DisCatSharp/Enums/Permission.cs

```diff
@@ -90,209 +90,321 @@ public enum Permissions : long
 		/// <summary>
 		/// Indicates no permissions given.
 		/// </summary>
-		[PermissionString("No permissions")]
+		[PermissionString("No Permissions")]
 		None = 0x0000000000000000,
 
 		/// <summary>
 		/// Indicates all permissions are granted
 		/// </summary>
-		[PermissionString("All permissions")]
+		[PermissionString("All Permissions")]
```

Comment:
I'm not exactly sure where these appear, but what's the rationale for using title case, as opposed to sentence case as is used in the official DC docs?

Reply:
Client-Side